packages/DropdownMenu/src/Item.styles.js

```diff
@@ -0,0 +1,52 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const itemStyles = css`
+  cursor: pointer;
+  display: block;
+  ${stylers.fontSize()};
```

Comment:
This should be first, before `cursor`.

Reply:
how come? Are we ordering things differently when using stylers?

Reply:
I have reordered it to be at the top though